build/Jenkinsfile

```diff
@@ -17,7 +17,12 @@ node() {
 
     stage('docker-build') {
         sh '''
-        docker build -t $docker_server/$docker_repo:$commit_id .
+        docker build -f ./Dockerfile.Build -t $docker_server/$docker_repo:$commit_id .
+		docker run --name=$docker_server/$docker_repo:$commit_id 
+		containerid=$(docker ps -aqf "name=$docker_server/$docker_repo:$commit_id .")
+		docker cp $containerid:/opt/telemetry-service.zip telemetry-service.zip
```

Comment:
Where is this telemetry-service.zip is being used after copying from the first container? Also, if the branch name is main or master tag the final build as `latest` as well.